chain/sync_test.go

```diff
@@ -833,3 +891,55 @@ func TestSyncCheckpointEarlierThanHead(t *testing.T) {
 	require.Equal(tu.t, p1Head, a.TipSet())
 	tu.assertBad(p1, b.TipSet())
 }
+
+func TestDrandNull(t *testing.T) {
```

Comment:
Seems to be failing on CI - https://app.circleci.com/pipelines/github/filecoin-project/lotus/14479/workflows/adc1f7a9-40b6-41f2-a72c-b9cb00f20c70/jobs/151579

Reply:
Yes, because of the tipset cache thing i mentioned above^